src/drt/src/TritonRoute.i

```diff
@@ -238,4 +238,12 @@ void check_drc_cmd(const char* drc_file, int x1, int y1, int x2, int y2)
   auto* router = ord::OpenRoad::openRoad()->getTritonRoute();
   router->checkDRC(drc_file, x1, y1, x2, y2);
 }
+
+void stack_vias(odb::dbBTerm* bterm,
```

Comment:
Is this just for testing or is it obsolete?  I don't any tcl caller.

Reply:
Obsolete. I just removed it.